src/utils/index.js

```diff
@@ -61,6 +61,7 @@ export const urlStatus = (tipUrl, verifiedUrls, blacklistedUrls) => {
   const url = twitterProfile || tipUrl;
 
   if (blacklistedUrls.some((u) => url.includes(u))) return 'blacklisted';
+  if (url.startsWith('http') && !url.startsWith('https:')) return 'not-secure';
```

Comment:
```suggestion
  if (!url.startsWith('https:')) return 'not-secure';
```

Shouldn't be enough? 
Looks good in general

Reply:
It avoids showing the `not-secure` status prematurely to the user when the URL is manually typed.